include/zephyr/bluetooth/hci_types.h

```diff
@@ -2905,6 +2935,26 @@ struct bt_hci_evt_le_req_peer_sca_complete {
 	uint8_t  sca;
 } __packed;
 
+#define BT_HCI_EVT_LE_TRANSMIT_POWER_REPORT     0x21
+
+/** Reason for Transmit power reporting.
+ */
+/* Local Transmit power changed. */
+#define	BT_HCI_LE_TX_POWER_REPORT_REASON_LOCAL_CHANGED 0
+/* Remote Transmit power changed. */
+#define	BT_HCI_LE_TX_POWER_REPORT_REASON_REMOTE_CHANGED 1
+/* HCI_LE_Read_Remote_Transmit_Power_Level command completed. */
+#define	BT_HCI_LE_TX_POWER_REPORT_REASON_READ_REMOTE_COMPLETED 2
```

Comment:
The HCI spec defines these in hexadecimal, so it looks a bit odd that you've decided to use decimal values. Just follow the same what the spec does:
<img width="706" alt="Screenshot 2023-11-27 at 11 57 02" src="https://github.com/zephyrproject-rtos/zephyr/assets/1032570/687eb7cf-9bf2-4710-a2bd-73920658f5d5">

Reply:
And one small thing: add an empty line between the defines and the struct